BobbyD.txt

```diff
@@ -1 +1 @@
-Hello! My name is Bobby. Three words that describe me are creative, friendly, smart. My favorite hobbies are painting, watching film/television, and arguing with people on the internet. My favorite animals are tree frogs.
```

Comment:
"arguing with people on the internet" Ahh I can relate to this on a spiritual level.